fiftyone/server/db/_mongo_lightning.py

```
@@ -0,0 +1,627 @@
+                field_path,
+                dataset,
+                -1,
+                is_frame_field,
+                limit=path.max_documents_search,
+            ),
+        ] + [
+            _match(field_path, v, limit=path.max_documents_search)
+            for v in (float("-inf"), float("inf"), float("nan"), None)
+        ]
+
+        def _resolve_float(results):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_resolve_float`

(ANN202)